src/style.css

```diff
@@ -0,0 +1,80 @@
+* {
+  box-sizing: border-box;
+}
+@font-face {
+  font-family: avenir;
+  src: url(../src/fonts/Avenir.ttc), url(../src/fonts/Avenir-Heavy.ttf), url(../src/fonts/Avenir-Book.ttf);
+}
+
+body {
+  margin-top: 20px;
+}
+
+.form:last-child {
+  margin-top: 20px;
+}
+
+.search-bar {
+  width: 100%;
+  font-family: avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  background-image: url(../src/images/Search.svg);
+  background-repeat: no-repeat;
+  border-radius: 4px;
+
+}
+
+.search-bar::placeholder {
+  font-family: avenir, Arial, Helvetica, sans-serif;
+}
+
+.search-bar:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search1 {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  background-size: 19px;
+  background-position-x: 26px ;
+  background-position-y: 50% ;
+}
+
+.search1:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  background-image: url(../src/images/Search.svg);
+  background-size: 19px;
+  background-repeat: no-repeat;
+  background-position-x: 26px ;
+  background-position-y: 50% ;
+}
+
+.search2 {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  background-size: 11px;
+  background-position-x: 13px ;
+  background-position-y: 50% ;
+}
+
+.search2:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  background-image: url(../src/images/Search.svg);
+  background-size: 11px;
+  background-repeat: no-repeat;
+  background-position-y: 50% ;
+  background-position-x: 13px ;
+}
+
+input:focus::placeholder {
+  font-weight: bold;
+}
```

Comment:
when in focus placeholder does not change in anyway
but you have to style placeholder in default mode